tests/snapshots/run_file__repeated_name_truncation.hvm.snap

```diff
@@ -0,0 +1,5 @@
+---
+source: tests/golden_tests.rs
+input_file: tests/golden_tests/run_file/repeated_name_truncation.hvm
+---
+λ* 2
```

Comment:
Was this change intentional?

Reply:
Yes, the other function had infinite expansion, so I changed the test to avoid that.

This still hits the expand problem where the `2` will sometimes be derefed and sometimes not.